### PR TITLE
Add in flags so notification and features don't show for GuestMode

### PR DESCRIPTION
**Pull request by timlevett**
+ No features for you GuestMode
+ No notifications for you GuestMode

### CODE REVIEW

**Comment by vertein** [comment]
:+1:

**Comment by Doug-Reed** [comment]
:+1:

File: uw-frame-components/portal/main/partials/header.html [hunk 1]
```diff
@@ -18,7 +18,7 @@
                          class="hidden-xs crest"
                          alt="{{portal.theme.crestalt}}">
                     <span class='title'>{{portal.theme.title}}</span>
-                    <!-- <span class="beta-logo">{{portal.theme.subtitle}}</span> -->
+                    <span class="beta-logo">{{portal.theme.subtitle}}</span>
```
Comment (apetro) [inline uw-frame-components/portal/main/partials/header.html]: Unrelated change?

**Comment by apetro** [comment]
:+1:

File: uw-frame-components/portal/main/partials/header.html (hunk 1 above)
Comment (timlevett) [inline uw-frame-components/portal/main/partials/header.html]: Yeah, forgot i did that. Not sure why that ever got commented out. We have the ability, thought it should be known.

http://goo.gl/XvTd0X
